apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/index.tsx

```diff
@@ -94,12 +101,12 @@ const commonFieldComponents: Record<string, FieldComponent> = {
   address: AddressField,
   text: TextField,
 };
-const Container = styled(Box).attrs(() => ({
+export const Container = styled(Box).attrs(() => ({
   alignItems: "center",
   fontSize: 4,
   pb: 4,
 }))``;
-const Info = styled(Box).attrs(() => ({
+export const Info = styled(Box).attrs(() => ({
```

Comment:
Do we need to export those components ? if not we should probably remove the export

Reply:
removed the export for `Container` and `Info` wasn't even used anymore so removed it altogether: 3172830c4a6a8069a2f75cb5771a3f8d88370112